package.json

```diff
@@ -29,7 +29,7 @@
     "uuid": "8.3.2"
   },
   "peerDependencies": {
-    "cypress": "7.6.0"
+    "cypress": "5.6.0"
```

Comment:
I've reverted this because we're still using 5.6.0 @klarstil @leichteckig

Reply:
Works for me, the 3.x branch is using the newer version of Cypress.